.travis.yml

```diff
@@ -90,16 +90,16 @@ jobs:
     - find . -not -path "./3rdparty/*" -and -not -path "./node_modules/*" -name "*.md" | xargs ./node_modules/.bin/markdownlint --config .markdownlintrc
 
   # Clippy linting
-  - &clippy-core
-    env: FEATURE=clippy-core
+  - &clippy
```

Comment:
It seems that this node identifier isn't used anywhere. Remove?

Reply:
I actually have no idea how borrowing works in YAML, so let's try removing it and see if anything breaks :-)